js-packages/@prestojs/ui-antd/src/styles.less

```diff
@@ -1 +0,0 @@
-@import './widgets/RangeWidget.less';
```

Comment:
we want to keep this file (just update import)

so in the end you can either import all styles (this file) or individual component styles